pcsx2/ImGui/FullscreenUI.cpp

```diff
@@ -998,6 +1060,26 @@ void FullscreenUI::DoChangeDiscFromFile()
 		std::string(Path::GetDirectory(s_current_disc_path)));
 }
 
+void FullscreenUI::RequestChangeDisc()
```

Comment:
To avoid the duplication here, you could do something like:
```cpp
void FullscreenUI::ConfirmShutdownIfMemcardBusy(std::function<void(bool)> callback)
{
  if (!MemcardBusy::IsBusy())
  {
    callback(true);
    return;
  }

  OpenConfirmMessageDialog(FSUI_ICONSTR(ICON_FA_SD_CARD, "WARNING: Memory Card Busy"),
    FSUI_STR("WARNING: Your memory card is still writing data. Shutting down now will IRREVERSIBLY DESTORY YOUR MEMORY CARD. It is strongly recommended to resume your game and let it finish writing to your memory card.\n\nDo you wish to shutdown anyways and IRREVERSIBLY DESTROY YOUR MEMORY CARD?"),
    std::move(callback));
}
```

Then e.g. here, you could do:
```cpp
ConfirmShutdownIfMemcardBusy([](bool result) {
				if (result)
					DoChangeDisc();

				ClosePauseMenu();
			});
```